libraries/AP_ADSB/AP_ADSB_uAvionix_UCP.cpp

```diff
@@ -253,7 +253,9 @@ void AP_ADSB_uAvionix_UCP::handle_msg(const GDL90_RX_MESSAGE &msg)
             _frontend.out_state.ctrl.x_bit = rx.decoded.transponder_status.x_bit;
         }
         run_state.last_packet_Transponder_Status_ms = AP_HAL::millis();
+#if HAL_ADSB_ENABLED
```

Comment:
This doesn't belong here. Maybe you meant if GCS enabled?

Reply:
Nope, these just don't belong at all.

Removed them.

Ta!